src/components/molecules/NavBar/NavBar.scss

```diff
@@ -150,56 +152,52 @@ $border-radius: 28px;
     }
   }
 }
-.nav-party-logo {
-  position: relative;
-  background-color: rgba($white, 0.08);
-  height: 60px;
-  padding: 0 50px 0 20px;
-  text-align: center;
-  cursor: pointer;
+
+.nav-schedule {
+  position: fixed;
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
   align-items: center;
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  transition: transform 400ms $transition-function;
-  border: none;
-  appearance: none;
+  justify-content: center;
+  top: 65px;
+  left: 0;
   color: $white;
+  width: $nav-container-width;
+  height: $nav-container-height;
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
```

Comment:
would be good to use an in-file variable here.

Reply:
fixed 57ea1af